frontend/web/components/release-pipelines/FlagActionDetail.tsx

```diff
@@ -41,7 +41,7 @@ const FlagActionDetail = ({
   const { data: segmentData } = useGetSegmentQuery(
     {
       id: `${actionBody.segment_id}`,
-      projectId: projectId,
+      projectId: `${projectId}`,
```

Comment:
out of curiosity, is it a type necessity ? Is RTK able to convert the number natively ?

Reply:
We can change the RTK parameter to accept number type but I'd rather have in a separate PR, because it would require updating the calls to useGetSegmentQuery.